virttest/cpu.py

```diff
@@ -731,7 +731,9 @@ def get_cpu_info(session=None):
         output = process.run(cmd, ignore_status=True).stdout_text.splitlines()
```

Comment:
I tried your fix for line 731, and it can also fix the problem on local host. Could you help also include this?